pom.xml

```diff
@@ -7,7 +7,7 @@
     <revision>2.0.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <betonquest.version>${revision}-DEV-UNOFFICIAL</betonquest.version>
-    <minecraft.api.version>1.18</minecraft.api.version>
+    <minecraft.api.version>1.19</minecraft.api.version>
```

Comment:
that is at the moment totally unwanted

Reply:
Understood. Feel free to cancel or keep this PR for your reference when you decide to update.